pkg/controller/complianceremediation/complianceremediation_controller.go

```diff
@@ -242,7 +242,8 @@ func getAppliedMcRemediations(r *ReconcileComplianceRemediation, rem *compliance
 		}
 
 		// OK, we've got an applied MC, add it to the list
-		appliedRemediations = append(appliedRemediations, &candidate.Spec.MachineConfigContents)
+		ar := candidate.Spec.MachineConfigContents
```

Comment:
Alternately you can use the indexing form of range to avoid reuse, eg: https://play.golang.org/p/xd7rBklWFaZ . (there's a good argument in favor of this form across the project to avoid accidents)

Reply:
I like this better, thank you for the suggestion.